views/content/token.md

```diff
@@ -571,7 +568,7 @@ If you add all the advanced options, this is how the final code should look like
 
         /* This unnamed function is called whenever someone tries to send ether to it */
         function () {
-            throw;     // Prevents accidental sending of ether
+            revert();     // Prevents accidental sending of ether
```

Comment:
This whole function can be removed, since now all functions are non payable by default